examples/assets/scripts/misc/camera-frame.mjs

```diff
@@ -0,0 +1,450 @@
+/* eslint-disable jsdoc/check-tag-names */
+
+import {
+    Script,
+    Color,
+    math,
+    RenderingParams,
+    CameraFrameOptions,
+    RenderPassCameraFrame,
+    FOG_NONE,
+    SSAOTYPE_NONE
+} from 'playcanvas';
+
+/** @enum {string} */
+const FogType = {
+    NONE: 'none',      // FOG_NONE
+    LINEAR: 'linear',  // FOG_LINEAR
+    EXP: 'exp',        // FOG_EXP
+    EXP2: 'exp2'       // FOG_EXP2
+};
+
+/** @enum {number} */
+const ToneMapping = {
+    LINEAR: 0,  // TONEMAP_LINEAR
+    FILMIC: 1,  // TONEMAP_FILMIC
+    HEJL: 2,    // TONEMAP_HEJL
+    ACES: 3,    // TONEMAP_ACES
+    ACES2: 4,   // TONEMAP_ACES2
+    NEUTRAL: 5  // TONEMAP_NEUTRAL
+};
+
+/** @enum {string} */
+const SsaoType = {
+    NONE: 'none',           // SSAOTYPE_NONE
+    LIGHTING: 'lighting',   // SSAOTYPE_LIGHTING
+    COMBINE: 'combine'      // SSAOTYPE_COMBINE
+};
+
+/** @enum {number} */
+const RenderFormat = {
+    RGBA8: 7,       // PIXELFORMAT_RGBA8
+    RG11B10: 18,    // PIXELFORMAT_111110F
+    RGBA16: 12,     // PIXELFORMAT_RGBA16F
+    RGBA32: 14      // PIXELFORMAT_RGBA32F
+};
+
+/** @interface */
+class Rendering {
+    /**
+     * @attribute
+     * @type {RenderFormat}
+     */
```

Comment:
If you add a `@title` tag and a single line description it will be used as the label and tooltip in the UI

```suggestion
    /**
     * Specifies the data format to use when rendering. 
     *
     * @attribute
     * @title Render Format
     * @type {RenderFormat}
     */
```